sql/moz-fx-data-shared-prod/contextual_services/event_aggregates/view.sql

```diff
@@ -2,6 +2,7 @@ CREATE OR REPLACE VIEW
   `moz-fx-data-shared-prod.contextual_services.event_aggregates`
 AS
 SELECT
-  *
+  * EXCEPT (advertiser),
+  LOWER(advertiser) as advertiser
```

Comment:
```suggestion
  * REPLACE(LOWER(advertiser) AS advertiser)
```

The REPLACE construction is a bit more elegant for this case where the field name is staying the same.

Reply:
@jklukas thanks! I saw REPLACE being used elsewhere, but when I looked it up in the bigquery documentation, I only found this [string function](https://cloud.google.com/bigquery/docs/reference/standard-sql/string_functions#replace) -- it made me wonder: is a different flavor of SQL being used here?

Reply:
REPLACE is overloaded, which can be confusing. The `* REPLACE` syntax is documented under SELECT: https://cloud.google.com/bigquery/docs/reference/standard-sql/query-syntax#modifiers_for_operator